mashina.Inc/src/main/java/com/epam/machine/Controller/util/loginCheck.java

```diff
@@ -0,0 +1,10 @@
+package com.epam.machine.Controller.util;
+
+public final class loginCheck {
+    public static boolean checkLog (String username, String password) {
+        if (username == null || password == null) throw new NullPointerException("No login or password");
```

Comment:
BadRequestException or IllegalArgumentException will more appropriate in this case

Reply:
And plz fix code style according to java code convention (p. 12)
https://www.oracle.com/technetwork/java/codeconventions-150003.pdf